src/content/docs/tips/script-tag-dynamic-imports.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Dynamic imports in module scripts
+description: How dynamic imports offer granular control over fetch and execution order
+astroRange: ^4.0.0
+---
+
+import ExampleLinks from '~/components/ExampleLinks.astro'
+
+
+Scripts with no attributes in Astro are automatically processed into [`type="module"`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) and all hoisted into one `.js` file that has static imports to other `.js` files with the imported code. It is possible to use [Dynamic module loading](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#dynamic_module_loading) instead of the default static imports, in order to gain more granular control over if/when the imported code is actually imported (downloaded).
```

Comment:
```suggestion
Scripts with no attributes in Astro are automatically processed into [`type="module"`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) and all hoisted into one `.js` file that has static imports to other `.js` files with the imported code. It is possible to use [dynamic module loading](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#dynamic_module_loading) instead of the default static imports, in order to gain more granular control over if/when the imported code is actually imported (downloaded).
```